cfme/configure/access_control.py

```diff
@@ -1261,20 +1274,24 @@ def is_displayed(self):
         )
 
 
-class EditTenantView(TenantForm):
+class EditTenantView(View):
     """ Edit Tenant View """
+    form = View.nested(TenantForm)
     save_button = Button('Save')
     reset_button = Button('Reset')
 
     @property
     def is_displayed(self):
         return (
-            self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name)
+            self.form.accordions.accesscontrol.is_opened and
+            any([self.form.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name),
+                self.form.title.text == 'Editing Project "{}"'.format(self.context['object'].name)])
         )
+        # used any() because `project` inherits `tenant` and edit page can have any one in name
 
 
-class Tenant(Updateable, Pretty, Navigatable):
+@attr.s
+class Tenant(BaseEntity):
```

Comment:
**REQUIRED:** Is the Tenant no longer Updateable?

Reply:
It is Updateable, added.